src/container/srv_target.c

```diff
@@ -2572,39 +2691,55 @@ ds_cont_tgt_ec_eph_query_ult(void *data)
 					}
 				}
 
-				if (!is_failed_tgts)
-					min_eph = min(min_eph, ec_eph->ce_ephs[i]);
+				if (!is_failed_tgts) {
+					min_ec_agg_eph = min(min_ec_agg_eph,
+							     ec_eph->cte_ec_agg_ephs[i]);
+					min_stable_eph = min(min_stable_eph,
+							     ec_eph->cte_stable_ephs[i]);
+				}
 			}
 
-			if (min_eph == 0 || min_eph == DAOS_EPOCH_MAX ||
-			    min_eph <= ec_eph->ce_last_eph) {
-				if (min_eph > 0 && min_eph < ec_eph->ce_last_eph)
-					D_ERROR("ignore for now "DF_X64" < "DF_X64
-						" "DF_UUID"\n", min_eph, ec_eph->ce_last_eph,
-						DP_UUID(ec_eph->ce_cont_uuid));
+			if (min_ec_agg_eph == 0 || min_ec_agg_eph == DAOS_EPOCH_MAX ||
+			    min_stable_eph == 0 || min_stable_eph == DAOS_EPOCH_MAX ||
+			    (min_ec_agg_eph <= ec_eph->cte_last_ec_agg_epoch &&
+			     min_stable_eph <= ec_eph->cte_last_stable_epoch)) {
+				if (min_ec_agg_eph > 0 && min_stable_eph > 0 &&
+				    (min_ec_agg_eph < ec_eph->cte_last_ec_agg_epoch ||
+				     min_stable_eph < ec_eph->cte_last_stable_epoch))
+					D_ERROR("ignore for now, min_ec_agg_eph "DF_X64" < "DF_X64
+						", or min_stable_eph "DF_X64" <"DF_X64
+						", "DF_UUID"\n",
+						min_ec_agg_eph, ec_eph->cte_last_ec_agg_epoch,
+						min_stable_eph, ec_eph->cte_last_stable_epoch,
+						DP_UUID(ec_eph->cte_cont_uuid));
 				else
 					D_DEBUG(DB_MD, "Skip eph "DF_X64"/"DF_X64
-						" "DF_UUID"\n", min_eph, ec_eph->ce_last_eph,
-						DP_UUID(ec_eph->ce_cont_uuid));
+						", "DF_X64"/"DF_X64", "DF_UUID"\n",
```

Comment:
suggest output of the log message includes description of what the values are, such as "Skip ec_agg_eph " for the first two, and then "stable_eph" for the second two values

Reply:
sure